appveyor.yml

```diff
@@ -18,6 +18,9 @@ environment:
 build_script:
   - cd ..
   - git clone --depth=1 https://github.com/xbmc/xbmc.git
+  - cd xbmc\project\BuildDependencies
+  - DownloadMingwBuildEnv.bat
```

Comment:
correct path is `tools/buildsteps/windows/(win32|x64)/download-msys2.bat`
the one you use only exists for backwards compatibility with v17 (jenkins win32 build)

Reply:
great. can i call them from top of tree ?

Reply:
can be called from any location

Reply:
I think you have to call the script with `sh` "parameter", otherwise it would open some additional Terminal window.